### ISSUE TITLE
Password-Managers fill the new Password field automatically.  

### ISSUE THREAD

**Issue by kzorluoglu** [chameleon-system/chameleon-system#633]
**Describe the bug**
When User one of many Password-Managers in your Browser installed, Password Manager fill the username and password field automatically. 

**Affected version(s)**
All Chameleon version or versions affected by this bug.

**To Reproduce**
1. Install one of many of Password Managers (LastPass, RoboForm etc.)
2. Login in CMS 
3. Save the Login Information via Password-Managers PopUp
4. Go to  Password Tab under Customers / Users > User
5. You see the saved Password automatically on the New Password field placed.

**Screenshots**

**Technical details**
The auto fill problem can be solved via input properties autocomplete="off" and autocomplete="new-password", but Password-Managers Extensions actually not filtering these autocomplete properties.

**Comment by UlrichKu** [comment]
That solution ("readonly but not readonly"; hacking the normal form logic) looks like a solution that invites other problems.

Sounds more like a problem of the password managers?

autocomplete is always ignored?

**Comment by kzorluoglu** [comment]
Right, the problems coming directly from the Password-Managers, they should have filtered the inputs via `autocomplete="off"` or `autocomplete="new-password"` tag, but actually not taking place **at the moment.**

I have basically testet for daily work:
-  After password change come the Save Pop-Ups from Managers
- Solution solving the problem with RoboForm 9.0.7.0 and LastPass 4.55.0

Please write, if you any test-case has.

**Comment by rnuernberg** [comment]
I would suggest to use `new-password` - I think most browsers will at least respect that.

**Comment by kzorluoglu** [comment]
i think, new-password makes more sense but, both of the Attributes `new-password` and `off` work problem-free with moderns browsers, i mean, which do not accept the auto fill attempts from  **native browsers**. 

But our problem is Password-Managers instead of browsers, the Password-Manager extensions, they do not filter the inputs with autocomplete attribute.

**Comment by UlrichKu** [comment]
But (at least in this case) this is such a low-level location that we do not want to introduce a Javascript workaround for a problem that may occur in special (faulty) browser extensions.

**Comment by rnuernberg** [comment]
I agree with @UlrichKu - there is no point creating a workaround for some random pwd manager extension that can introduce new problems. As far as I understand, `new-password` more correctly represents our intention here - so I would go with that.
